### PR TITLE
Hn ah ticket post

### PR DESCRIPTION
**Pull request by a-holliday**
Daos filled out, second pull request.

### CODE REVIEW

File: BugTracker/src/main/java/BugTracker/dao/EmployeeDao.java [hunk 1]
```diff
@@ -24,4 +26,13 @@
 	public void deleteEmployee(Employee employee);
 
 	public Employee findAssigned();
+
+	List<Employee> leaderBoard();
```
Comment (kmilstein) [inline BugTracker/src/main/java/BugTracker/dao/EmployeeDao.java:30]: Method already exists

File: BugTracker/src/main/java/BugTracker/dao/EmployeeDao.java [hunk 2]
```diff
@@ -24,4 +26,13 @@
...
+
+	/**
+	 * readAllEmployees returns all the employees in the database
+	 * @returns List<Employee> 
+	 * @author Acacia and Hannah
+	 */
+	List<Employee> readAllEmployees();
```
Comment (JoeDKoal94) [inline BugTracker/src/main/java/BugTracker/dao/EmployeeDao.java:37]: please delete method, already have the method for experience

File: BugTracker/src/main/java/BugTracker/dao/EmployeeDaoImpl.java [hunk 3]
```diff
@@ -110,4 +137,24 @@ public Employee findAssigned() {
 		return employee;
 	}
 
+	@Override
```
Comment (kmilstein) [inline BugTracker/src/main/java/BugTracker/dao/EmployeeDaoImpl.java:140]: Already implemented

File: BugTracker/src/main/java/BugTracker/dao/EmployeeDaoImpl.java [hunk 4]
```diff
@@ -110,4 +137,24 @@ public Employee findAssigned() {
...
+	 */
+	@Override
+	public List<Employee> readAllEmployees() {
+		Session sess = sessionFactory.openSession();
+		
+		List<Employee> employeeList = (List<Employee>) sess.createQuery("from Employee");
+		
+		return employeeList;
+	}
+
```
Comment (JoeDKoal94) [inline BugTracker/src/main/java/BugTracker/dao/EmployeeDaoImpl.java:159]: have the method for leaderboard, delete this.

File: BugTracker/src/main/java/BugTracker/dtos/TicketDto.java [hunk 5]
```diff
@@ -154,9 +142,26 @@ public boolean equals(Object obj) {
 
 	@Override
 	public String toString() {
-		return "TicketDto [openerId=" + openerId + ", name=" + name + ", created=" + created + ", statusId=" + statusId
-				+ ", priorityId=" + priorityId + ", difficultyLevel=" + difficultyLevel + ", body=" + body + "]";
+		return "TicketDto [openerId=" + openerId + ", name=" + name + ", statusId=" + statusId + ", priorityId="
+				+ priorityId + ", difficultyLevel=" + difficultyLevel + ", body=" + body + "]";
 	}
+
```
Comment (kmilstein) [inline BugTracker/src/main/java/BugTracker/dtos/TicketDto.java:148]: Why does this have so many spaces!!!! ;-;

**Review by kmilstein — CHANGES_REQUESTED** [review]

**Review by kmilstein — APPROVED** [review]

**Pull request merged**